### ISSUE TITLE
React Aria warnings on radio fields

### ISSUE THREAD

**Issue by McManning** [osuresearch/ui#61]
React-Aria spams the console with the following warning:

```
If you do not provide children, you must specify an aria-label for accessibility
```

This is traced from the `useRadio` hook used in YesNo, DefaultGroupItemSlot, and other fields. It looks like since my implementation does not explicitly send a label or children to the hook, it's complaining. I do still render things correctly down the component tree to meet a11y but, unfortunately, the hook has other opinions on how things should be done.

**Comment by McManning** [comment]
Also this is a major performance impact on apps, since each visible radio will log a warning each re-render.
